packages/contentstack-command/package.json

```diff
@@ -17,7 +17,7 @@
     "format": "eslint src/**/*.ts --fix"
   },
   "dependencies": {
-    "@contentstack/cli-utilities": "^1.0.2",
+    "@contentstack/cli-utilities": "^1.0.3",
```

Comment:
@abhinav-from-contentstack Update the package version.

And need to update all the location where this package is used.

CC @netrajpatel

Reply:
done